hs_discover/views.py

```diff
@@ -70,47 +70,47 @@ def get(self, request, *args, **kwargs):
             qs = request.query_params
             filters = json.loads(qs.get('filter'))
             # filter values expect lists, for example discoverapi/?filter={"owner":["Firstname Lastname"]}
-            if filters.get('author'):
+            if filters.get('author') and len(filters['author']) > 0:
```

Comment:
you could simplify to `len(filters.get('author', '')) > 0`

Reply:
filters.get('author') can return None, crashing this.

Reply:
I'm going nuts today. I really can't handle this change right now.